README.md

```diff
@@ -0,0 +1,428 @@
+Drag and Drop XBlock v2
+=======================
+
+This XBlock implements a friendly drag-and-drop style question, where
+the student has to drag items on zones on a target image.
+
+The editor is fully guided. Features include:
+
+* custom target image
+* free target zone positioning and sizing
+* custom zone labels
+* ability to show or hide zone borders
+* custom text and background colors for items
+* image items
+* items prompting for additional (numerical) input after being dropped
+* decoy items that don't have a zone
+* feedback popups for both correct and incorrect attempts
+* introductory and final feedback
+
+The XBlock supports progressive grading and keeps progress across
+refreshes. All checking and record keeping is done on the server side.
+
+The following screenshot shows the Drag and Drop XBlock rendered
+inside the edX LMS before the user starts solving the problem:
+
+![Student view start](https://raw.githubusercontent.com/edx-solutions/xblock-drag-and-drop-v2/c955a38dc3a1aaf609c586d293ce19b282e11ffd/doc/img/student-view-start.png)
+
+This screenshot shows the XBlock after the student successfully
+completed the Drag and Drop problem:
+
+![Student view finish](https://raw.githubusercontent.com/edx-solutions/xblock-drag-and-drop-v2/c955a38dc3a1aaf609c586d293ce19b282e11ffd/doc/img/student-view-finish.png)
+
+Installation
+------------
+
+Install the requirements into the Python virtual environment of your
+`edx-platform` installation by running the following command from the
+root folder:
+
+```bash
+$ pip install -r requirements.txt
+```
+
+Theming
+-------
+
+The Drag and Drop XBlock ships with an alternate theme called "Apros"
+that you can enable by adding the following entry to `XBLOCK_SETTINGS`
+in `lms.env.json`:
+
+```json
+        "drag-and-drop-v2": {
+            "theme": {
+                "package": "drag_and_drop_v2",
+                "locations": ["public/themes/apros.css"]
+            }
+        }
+```
+
+You can use the same approach to apply a custom theme:
+
+`"package"` can refer to any Python package in your virtualenv, which
+means you can develop and maintain your own theme in a separate
+package. There is no need to fork or modify this repository in any way
+to customize the look and feel of your Drag and Drop exercises.
```

Comment:
@pdesjardins Thanks for this. Searching the code base I found additional occurrences of the word "exercise(s)"; changed all of them to "problem(s)" in https://github.com/edx-solutions/xblock-drag-and-drop-v2/commit/ed3e323cffae49523fbe62f5056c2207ae38a0df. In an effort to further standardize the terminology, I also replaced remaining occurrences of "question(s)" with "problem(s)" wherever possible.